apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt_actions.erl

```diff
@@ -370,18 +370,16 @@
         },
         ssl => #{
             order => 11,
-            type => string,
-            required => false,
-            default => <<"off">>,
-            enum => [<<"on">>, <<"off">>],
+            type => boolean,
```

Comment:
In the data import and export functions, this line raises format compatibility issues.
We should fix the code over there in the next sprint, I've documented it in Jira first

see: https://emqx.atlassian.net/browse/EMQX-336